src/Python/SimpleOperations/UniformRandomNumber.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import vtk
+import vtkmath
+def main():
+    numRand = 3;
+    #Generate numRand random numbers from a uniform distribution between 0.0 and 2.0
+    for i in range(0,numRand):
+    	i = vtkmath.random.uniform(0.0,2.0)
+    	print(i)
```

Comment:
random is no longer recommended use [vtkMinimalStandardRandomSequence](https://vtk.org/doc/nightly/html/classvtkMinimalStandardRandomSequence.html) and [vtkBoxMuellerRandomSequence](https://vtk.org/doc/nightly/html/classvtkBoxMuellerRandomSequence.html) I will be removing random from existing code. Do look at existing python examples to see how it is used.

Reply:
@ajpmaclean Sure!!